code/drasil-printers/Language/Drasil/TeX/Helpers.hs

```diff
@@ -105,26 +108,18 @@ genSec d
 -- For references
 ref, sref, hyperref, externalref, snref :: String -> D -> D
 sref         = if numberedSections then ref else hyperref
-ref      t   = custRef (t ++ "~\\ref")
-hyperref t x = command0 "hyperref" <> sq x <> br (pure (text (t ++ "~")) <> x)
+ref      t x = pure (text $ t ++ "~\\ref") <> br x
+hyperref t x = command1oD "hyperref" (Just x) (pure (text (t ++ "~")) <> x)
```

Comment:
This use of `command1oD` (and the next) both seem a little silly, since they hard-code a `Just`. That function is really meant to be used when you have a dynamic `Maybe`. There ought to be a more direct combinator that does the same job that should be used here (and in `snref`).

Reply:
Fixed in dac98cdacbf2ba208917c12c07abd48f238b8d7b